rllib/core/models/configs.py

```diff
@@ -107,6 +108,51 @@ def build(self, framework: str = "torch") -> Model:
             return TfMLPHead(self)
 
 
+@ExperimentalAPI
+@dataclass
+class CNNEncoderConfig(ModelConfig):
+    """Configuration for a convolutional network.
+
+    See ModelConfig for usage details.
+
+    Attributes:
+        input_dims: The input dimension of the network. It cannot be None.
+        filter_specifiers: A list of lists, where each element of an inner list
+            contains elements of the form
+            `[number of filters, [kernel width, kernel height], stride]` to specify a
+            convolutional layer stacked in order of the outer list.
+        filter_layer_activation: The activation function to use after each layer (
+            except for the output).
+        output_activation: The activation function to use for the output layer.
+        output_dim: The output dimension are [BATCH, output_dim]. We append a final
```

Comment:
I am confused about mentioning BATCH here all of a sudden. All of this dims have an implicit batch in them, right? Do you want to say that in the end we will add a flatten layer to obtain output_dim features?

Reply:
Yeah, I removed the mentioning of BATCH. Thanks!